packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/CSharpType.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I wonder what did we change here?

Reply:
Probably the UTF-8 BOM at the start of the file